kernel/packages/shared/web3.ts

```diff
@@ -138,6 +138,27 @@ export async function fetchOwner(url: string, name: string) {
   }
 }
 
+export async function fetchENSOwnersContains(url: string, name: string, maxResults: number) {
+  const query = `
+    query GetOwner($name: String!) {
```

Comment:
We query the graph with this and that's it? We have our username search? 🎉 

Maybe it's nice to have this method documented as well? As `fetchENSOwnersContains` name its a bit opaque.